src/app/dynamic-forms/fields/child-form-table/child-form-table.component.html

```diff
@@ -49,7 +49,13 @@
       </tbody>
     </table>
   </div>
-  <p *ngIf="!childForm.childFormRowValues?.length">
-    There is no data in this table. You can use the add button above to get started.
+  <p *ngIf="childForm.childFormLinkId == null || childForm.childFormParentId == null ;else noRecord"> 
```

Comment:
This should use the `!` syntax instead of `== null`.

Amend to:

`<p ``*ngIf="!childForm.childFormLinkId || !childForm.childFormParentId; else noRecord">```

Reply:
![image](https://github.com/cinchy-co/meta-forms-app-experience/assets/119648879/ec5f5074-4e39-43eb-b783-c1f8444b003e)